test/v2/unit/recovery.ts

```diff
@@ -382,7 +381,7 @@ describe('Recovery:', function() {
         ignoreAddressTypes: ['p2wsh', 'p2shP2wsh']
       });
 
-      recovery.transactionHex.should.equal('0400008085202f89010f89e651a4d827d82dbe164cae6c09c21435b950a395e35afae813a1f775497500000000fc00473044022051814041484343dc52fa435be57a892a9084be2672801ae319328149ad96618e022029d36c49865b9df3b4c5820aec7ed44ba5ad787bca997d42313197c62b43473e01473044022027c7c2f527c2f5e3794e2e51611611e5f4b341e1736b0fde660cb4425ba9220702202f4e5ae7047bb2615680fa92ff29397d51a3da8acc3dc2f930899c613c8d33fe014c6952210222dba86781026f53d30be3bd2d07678c61926cb52c0de52b6ecef3d5c96e32fa2102b8a369ca2ef0d202b342fe3742585468813bebf821856fa4c2e90337bcee1d5e2102dcb29b842c2bb5e1efcab6483db61ab06bc08cb8c2667399bb1b5fb826a841a153aeffffffff01b03ec9010000000017a91470391ef30163f580806ee8a5f0aacc724e7f68558700000000000000000000000000000000000000');
+      recovery.transactionHex.should.equal('0400008085202f89010f89e651a4d827d82dbe164cae6c09c21435b950a395e35afae813a1f775497500000000fdfd00004830450221009b7e8045e3c45af7ddf218d12d40c560990056a92b73ae415b8f9becca4aa52102203bc2d9de4850330cfacf41a2db618b281ce1b612a8bc3e469b73e190bfef45db0147304402205642ada15fee4d0de20766b831128fc4ec4db0f4485912f682865dd3eceae7f50220149ab8e6c43c4e4a8493c20632cd29dc19bc05d380ce3df986fb8705c199fd1d014c6952210222dba86781026f53d30be3bd2d07678c61926cb52c0de52b6ecef3d5c96e32fa2102b8a369ca2ef0d202b342fe3742585468813bebf821856fa4c2e90337bcee1d5e2102dcb29b842c2bb5e1efcab6483db61ab06bc08cb8c2667399bb1b5fb826a841a153aeffffffff01643ac9010000000017a91470391ef30163f580806ee8a5f0aacc724e7f68558700000000000000000000000000000000000000');
```

Comment:
I've been running a script to verify that these output tx hexes are validly signed, and all have been good up until this one, which has thrown the following error 
```
Unhandled rejection RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 353. Received 355
    at boundsError (internal/buffer.js:53:9)
    at Buffer.readUInt32LE (internal/buffer.js:109:5)
    at readUInt32 (/Users/danny/node_modules/bitgo/node_modules/bitgo-utxo-lib/src/transaction.js:96:20)
    at Function.Transaction.fromBuffer (/Users/danny/node_modules/bitgo/node_modules/bitgo-utxo-lib/src/transaction.js:280:14)
    at Function.Transaction.fromHex (/Users/danny/node_modules/bitgo/node_modules/bitgo-utxo-lib/src/transaction.js:350:22)
```